docker/test/integration/minifi/validators/SingleFileContentHashValidator.py

```diff
@@ -0,0 +1,43 @@
+import logging
+import os
+
+from os import listdir
+from os.path import join
+
+from .FileOutputValidator import FileOutputValidator
+from ..core.HashUtils import md5
+
+class SingleFileContentHashValidator(FileOutputValidator):
+    """
+    Validates the content of a single file in the given directory.
+    """
+
+    def __init__(self, expected_md5_hash, subdir=''):
+        self.valid = False
+        self.expected_md5_hash = expected_md5_hash
+        self.subdir = subdir
+
+    def validate(self):
+        self.valid = False
+        full_dir = os.path.join(self.output_dir, self.subdir)
+        logging.info("Output folder: %s", full_dir)
+
+        if not os.path.isdir(full_dir):
+            return self.valid
+
+        listing = listdir(full_dir)
+        if listing:
+            for l in listing:
```

Comment:
I'm not a fan of one-char variable names, especially `l` (and `O` etc) -- this could be `file_name`, for example

Reply:
Updated